t/udpfw.c

```diff
@@ -174,6 +180,7 @@ static void emit_queue(struct queue_t *q, int up, int64_t now)
 
 static int read_queue(struct queue_t *q, struct connection_t *conn, int64_t now)
 {
+    static uint64_t packet_num = 0;
```

Comment:
We might want to consider making `packet_num` a member of `struct connection_t`, because then we will see every Nth packet of of a connection (identified by 5-tuple) being dropped.

In the current approach, all of the received packets will be considered as the first (i.e. 0th) packet, because this function reads only one packet, and packet_num is always set to zero.

Reply:
Yes, I missed that, good point. I've fixed it now.